lib/ex_machina.ex

```diff
@@ -96,6 +100,35 @@ defmodule ExMachina do
   end
 
   @doc """
+  Builds a factory with the passed in factory_name and returns its fields
+
+  This is only for use with Ecto models.
```

Comment:
Maybe add what it will do at the end? `Will return a map with the fields and virtual fields, but without the Ecto metadata and associations.`?